cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -152,8 +154,12 @@ var (
 		Short:             "List available frequency plans for end devices",
 		PersistentPreRunE: preRun(),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if !config.GatewayServerEnabled {
+				return errGatewayServerDisabled
+			}
+
 			baseFrequency, _ := cmd.Flags().GetUint32("base-frequency")
-			gs, err := api.Dial(ctx, config.NetworkServerGRPCAddress)
+			gs, err := api.Dial(ctx, config.GatewayServerGRPCAddress)
```

Comment:
:eye:

Reply:
This indeed doesn't seem quite right. Device frequency plans come from the network server, not the gateway server.

Reply:
Device frequency plans indeed, gateway frequency plans come from GS. Will fix that